modules/consentManagement.js

```diff
@@ -470,6 +470,7 @@ function exitModule(errMsg, hookConfig, extraArgs) {
 export function resetConsentData() {
   consentData = undefined;
   userCMP = undefined;
+  consentTimeout = undefined;
```

Comment:
note i added this line because it not getting reset in USP was causing issues with the tests; it seemed simple to fix here as well at the same time.